apps/remix-ide-e2e/src/tests/proxy.test.ts

```diff
@@ -70,12 +70,13 @@ module.exports = {
     browser
       .openFile('myTokenV1.sol')
       .clickLaunchIcon('solidity')
-      .pause(2000)
+      .pause(5000)
```

Comment:
Don't add pauses, if you can use assertions to see if the element is present.same for the other pause.

Reply:
Ok. Thanks